.github/workflows/channels-ci.yml

```diff
@@ -83,6 +83,16 @@ jobs:
           npm run mmjstool -- i18n clean-empty --webapp-dir ./src --mobile-dir /tmp/fake-mobile-dir --check
           npm run mmjstool -- i18n check-empty-src --webapp-dir ./src --mobile-dir /tmp/fake-mobile-dir
           rm -rf tmp
+      - name: ci/lint
+        working-directory: webapp/boards
+        run: |
+          npm run i18n-extract
+          git --no-pager diff --exit-code i18n/en.json || (echo "Please run \"cd webapp/boards && npm run i18n-extract\" and commit the changes in webapp/boards/i18n/en.json." && exit 1)
```

Comment:
I take it the `git diff` is sort of like what we're doing above by copying the `en.json` to a temp folder, regenerating `en.json`, and then comparing them above? I wonder if that's an option for Channels once the i18n package is set up

Reply:
Yes, that should be possible. Formatjs/cli does the cleanup at the same time as extraction which makes that possible. It could maybe even be done now with the current tooling via `npm run i18n-extract && npm run i18n-clean-empty` and then the git diff check.